.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+    "workbench.colorCustomizations": {
+        "activityBar.background": "#1A3130",
+        "titleBar.activeBackground": "#254543",
+        "titleBar.activeForeground": "#F6FAFA"
+    }
+}
```

Comment:
Should this be checked into the repo?

Reply:
Done in 2bddea109ad2cea8889ee3c8c09c210a4c088cde

Reply:
No it should not,